Dockerfile

```diff
@@ -41,16 +32,16 @@ WORKDIR /app
 RUN apk add --no-cache bash ca-certificates tzdata
 
 # See README.md and config.default.yml for all config options
-ENV ENVIRONMENT prod
-ENV WAKAPI_DB_TYPE sqlite3
-ENV WAKAPI_DB_USER ''
-ENV WAKAPI_DB_PASSWORD ''
-ENV WAKAPI_DB_HOST ''
-ENV WAKAPI_DB_NAME=/data/wakapi.db
-ENV WAKAPI_PASSWORD_SALT ''
-ENV WAKAPI_LISTEN_IPV4 '0.0.0.0'
-ENV WAKAPI_INSECURE_COOKIES 'true'
-ENV WAKAPI_ALLOW_SIGNUP 'true'
+ENV ENVIRONMENT=prod \
+    WAKAPI_DB_TYPE=sqlite3 \
+    WAKAPI_DB_USER='' \
+    WAKAPI_DB_PASSWORD='' \
+    WAKAPI_DB_HOST='' \
+    WAKAPI_DB_NAME=/data/wakapi.db \
+    WAKAPI_PASSWORD_SALT='' \
+    WAKAPI_LISTEN_IPV4='0.0.0.0' \
+    WAKAPI_INSECURE_COOKIES='true' \
+    WAKAPI_ALLOW_SIGNUP='true'
 
```

Comment:
I realized the container crashes with a confusing error message when using SQLite and not mounting the volume to `/data`. While using a volume is highly encouraged for data persistence, the container should still be able to without one. Could you please add this line here:

```
RUN mkdir /data
```

Reply:
Yes, I came across this also. 
I was thinking that it would be better to verify this in code, as it could be a valid occurrence outside Docker images.

Reply:
I think outside of the container it's fine, because there the default configuration doesn't result in a crash.

Default config for the SQLite file is `wakapi_db.db` (current directory), which is always available.

But in the container, the default path is `/data/wakapi.db`, which is not available by default and you would have to explicitly change the environment variable to fix this.